src/ansys/fluent/core/services/datamodel_se.py

```diff
@@ -661,23 +662,24 @@ def _create_command_arguments(self):
         response = self.service.create_command_arguments(request)
         return response.commandid
 
-    def _get_static_info(self):
+    def _generate_static_info(self):
         request = DataModelProtoModule.GetStaticInfoRequest()
         request.rules = self.rules
         response = self.service.get_static_info(request)
-        return response.info
+        PyCommand._stored_static_info = response.info
```

Comment:
@prmukherj 

this code needs to be changed so that the above container is a dict where each key is a rules name. Otherwise we are trying to share the same info for every rules name.

Also, new doesn't really need to change. get_static_info can encapsulate the instantiation of each static info.

Reply:
Thank you for the feedback @seanpearsonuk. It has been implemented, please let me know what you think of the current implementation.